.github/workflows/pr_open.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           name: control-dist
           path: cmd/control/
       - name: run tests
-        run: go test -tags json1 -v -coverpkg ./... -coverprofile coverage.out -covermode atomic ./...
```

Comment:
Can you add why/what this is in the PR description please ?

Reply:
I removed that because the tests give so many lines than the CI does not show all and we don't see errors sometimes.